docs: add doc for logging (#2578)
aws/copilot-cli: site/content/docs/include/common-svc-fields.en.md

<div class="separator"></div>

<a id="logging" href="#logging" class="field">`logging`</a> <span class="type">Map</span>  

> The logging section contains parameters for log configuration options to send to a firelens log driver for the container.

Reviewer: ```suggestion
The logging section contains log configuration parameters for your container's FireLens log driver.
```

Author: done!

<div class="separator"></div>

<a id="logging" href="#logging" class="field">`logging`</a> <span class="type">Map</span>  

> The logging section contains log configuration parameters for your container's FireLens log driver. This will create a logging [sidecar](../developing/sidecars.en.md#sidecar-patterns).

Reviewer 2: Maybe
```suggestion
The logging section contains parameters for log configuration options to send the log of the container using [Firelens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html) (see examples [here](../developing/sidecars.en.md#sidecar-patterns)).
```

<span class="parent-field">logging.</span><a id="logging-image" href="#logging-image" class="field">`image`</a> <span class="type">Map</span>  
Optional. The Fluent Bit image to use. Defaults to `amazon/aws-for-fluent-bit:latest`.

<span class="parent-field">logging.</span><a id="logging-destination" href="#logging-destination" class="field">`destination`</a> <span class="type">Map</span>  
Optional. The configuration options to send to the FireLens log driver.

<span class="parent-field">logging.</span><a id="logging-enableMetadata" href="#logging-enableMetadata" class="field">`enableMetadata`</a> <span class="type">Map</span>  

> Optional. Whether to include ECS metadata in logs. Defaults to true.

Reviewer 2: ```suggestion
Optional. Whether to include ECS metadata in logs. Defaults to `true`.
```
